.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,8 +12,8 @@
 
 ## Checklist
 <!--- If a checkbox below is not applicable, then please DELETE it rather than leaving it unchecked -->
-- [ ] I have updated `CHANGELOG.next.toml` if I made changes to the smithy-rs codegen or runtime crates
-- [ ] I have updated `CHANGELOG.next.toml` if I made changes to the AWS SDK, generated SDK code, or SDK runtime crates
+- [ ] For changes to the smithy-rs codegen or runtime crates, I have created a changelog entry Markdown file in the `.changelog` directory, specifying "client," "server," or both in the `applies_to` key.
```

Comment:
Is it possible to put a `README.md` in the changelog folder that explains how to manually create an entry AND points to "how to install and use the changelogger CLI"?

Specifically, how should users name their changelog if they don't use the CLI?

Reply:
Docs augmented in f08d91a.

> Is it possible to put a README.md in the changelog folder 

Do you mean `.changelog` folder? Since the files in that directory are used by automation, I wouldn't put `README.md` there.

> explains how to manually create an entry
> Specifically, how should users name their changelog if they don't use the CLI?

For these, I elaborated in `CONTRIBUTING.md` and in the `changelogger`'s `README.md`.